### ISSUE TITLE
interesting project

### ISSUE THREAD

**Issue by DoctorRay2** [foulman/Fantasy#54]
Hello! I've looked for a fantasy mod for CDDA for quite some time now, and stumbled across this project, do you guys have any discord or other forum for this? I would be interested in helping out if needed. I have over 10 years experience with spriting and coding.

**Comment by MNG-cataclysm** [comment]
There is a Discord channel that is used to talk about the project you can find [here](https://discord.gg/Y4WcSZB2), but I'm the only one who currently maintains it. There is one other person who wanted to work on it and made a branch, but I have no clue how that's going for them. I can't expand a total expansion mod alone, and I have some other commitments made on the main Cataclysm branch, so I just keep the mod updates and free from most errors.

**Comment by Alicyl** [comment]
> I can't expand a total expansion mod alone, and I have some other commitments made on the main Cataclysm branch, so I just keep the mod updates and free from most errors.

I hope more people will assist you with this project. 

I've been looking for a fantasy and medieval conversion/total expansion mod for a while now, and now that I finally found one—yours, I would love to see it gain more support and come to full fruition. Even though it's not being as maintained as the other popular mods, is it currently safe to use with 0.G (Steam)?

**Comment by MNG-cataclysm** [comment]
@Alicyl Yes, I updated it to work with 0.G, but it will break on the experimental builds. If it doesn't work for you, just let me know.

I don't know if 0.G on Steam and 0.G on GitHub have different changes, but it works for 0.G on GitHub.

**Comment by foulman** [comment]
> @Alicyl Yes, I updated it to work with 0.G, but it will break on the experimental builds. If it doesn't work for you, just let me know.
> 
> I don't know if 0.G on Steam and 0.G on GitHub have different changes, but it works for 0.G on GitHub.

Well well, I really should check my email more often, I just saw this!
I never would have imagined anyone would still be bothering with anything on here, much less keeping it up to date!

**Comment by Alicyl** [comment]
> Well well, I really should check my email more often, I just saw this! I never would have imagined anyone would still be bothering with anything on here, much less keeping it up to date!

Glad you did because we appreciate the work you and the collaborators did to make and maintain this mod—I most certainly do as someone who loves Low/High Fantasy over Sci-Fi. ❦

**Comment by foulman** [comment]
> Glad you did because we appreciate the work you and the collaborators did to make and maintain this mod—I most certainly do as someone who loves Low/High Fantasy over Sci-Fi. ❦

Honestly I don't think I could take much credit for this, I got it started certainly, but when I started having a hard time and needed to step away, several others have taken over and contributed greatly, @MNG-cataclysm I gave pretty much full control because they were interested. I'm glad to see there's still something here that's usable though!
I don't think I even have much of a clue what is in it at this point! 😆

**Comment by chaosvolt** [comment]
I keep wanting to help with it someday but stuff has always come up.

**Comment by Aptronymist** [comment]
> I keep wanting to help with it someday but stuff has always come up.

It's the dawn of AI (okay, more like late morning of AI), so why not "work smart, not hard!" it? 😜
Oh nevermind, I'll do it myself, but I'm starting with my other mods first...
